src/LcdButtons.cpp

```diff
@@ -10,11 +10,11 @@ LcdButtons::LcdButtons(byte pin, LcdMenu* lcdMenu) {
     _analogPin = pin;
     _lastKeyChange = 0;
 
-    _newKey = -1;
-    _lastNewKey = -2;
+    _newKey = btnNONE;
+    _lastNewKey = btnNONE;
 
-    _currentKey = -1;
-    _lastKey = -2;
+    _currentKey = btnNONE;
+    _lastKey = btnNONE;
```

Comment:
I need to check about this, but I think last is initialized different from current so that the first comparison fails (if (current!=last))
I think this would miss the very first key press after boot up.